core/src/test/scala/integration/kafka/api/SaslScramSslEndToEndAuthorizationTest.scala

```diff
@@ -42,7 +42,18 @@ class SaslScramSslEndToEndAuthorizationTest extends SaslEndToEndAuthorizationTes
   override def setUp(): Unit = {
     super.setUp()
     // Create client credentials after starting brokers so that dynamic credential creation is also tested
-    createScramCredentials(zkConnect, JaasTestUtils.KafkaScramUser, JaasTestUtils.KafkaScramPassword)
-    createScramCredentials(zkConnect, JaasTestUtils.KafkaScramUser2, JaasTestUtils.KafkaScramPassword2)
+    createScramCredentialWithScramAdminClient(JaasTestUtils.KafkaScramUser, JaasTestUtils.KafkaScramPassword)
+    createScramCredentialWithScramAdminClient(JaasTestUtils.KafkaScramUser2, JaasTestUtils.KafkaScramPassword2)
+  }
+
+  private def createScramCredentialWithScramAdminClient(user: String, password: String) = {
```

Comment:
So the reason for not using the `SaslSertup#createScramCredentials` method here is because we want the admin client itself to be authenticated with SCRAM?

Reply:
It was a goal to eliminate all SCRAM credential creation via ZooKeeper where possible.  The only places that do so after this PR are when credentials have to be created before the brokers are started (i.e. when the inter-broker security protocol is SASL/SCRAM).  This code used to create the credential directly via ZooKeeper, but since it occurs after the brokers start it can use the admin client.